### PR TITLE
Connecting personal archives added

### PR DESCRIPTION
**Pull request by Apoc70**
Added examples about identifying and connecting personal archive mailboxes, and a note on connecting a personal archive to a different primary mailbox.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeServer/recipients/disconnected-mailboxes/connect-disabled-mailboxes.md [hunk 1]
```diff
@@ -102,6 +102,21 @@ Connect-Mailbox -Identity "Corporate Shared Mailbox" -Database "Mailbox Database
 > [!NOTE]
 > If you don't include the _Alias_ parameter when you run the **Connect-Mailbox** cmdlet, the value specified in the _User_ or _LinkedMasterAccount_ parameter is used to create the email address alias for the reconnected mailbox.
 
+This example connects a personal archive to the primary mailbox using the mailbox Guid stored in mailbox database DB01.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/recipients/disconnected-mailboxes/connect-disabled-mailboxes.md]:
```suggestion
This example connects a personal archive to the primary mailbox using the mailbox GUID stored in mailbox database DB01.
```

File: Exchange/ExchangeServer/recipients/disconnected-mailboxes/connect-disabled-mailboxes.md [hunk 2]
```diff
@@ -77,9 +77,9 @@ The following procedure shows how to connect a disabled user mailbox. You can al
 
     Exchange will reconnect the disabled mailbox to the corresponding user account.
 
-## Use the Exchange Management Shell to connect a disabled mailbox
+## Use the Exchange Management Shell to connect a disabled mailbox or personal archive
 
-Use the **Connect-Mailbox** cmdlet in the Exchange Management Shell to connect a user account to a disabled mailbox. You have to specify the type of mailbox that you're connecting. The following examples show the syntax for reconnecting user, linked, and shared mailboxes.
+Use the **Connect-Mailbox** cmdlet in the Exchange Management Shell to connect a user account to a disabled mailbox. You have to specify the type of mailbox that you're connecting. The following examples show the syntax for reconnecting user, linked, shared, and archive mailboxes.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/recipients/disconnected-mailboxes/connect-disabled-mailboxes.md]:
```suggestion
Use the **Connect-Mailbox** cmdlet in the Exchange Management Shell to connect a user account to a disabled mailbox. You have to specify the type of mailbox that you're connecting. The following examples show the syntax for reconnecting user, linked, shared, and archived mailboxes.
```

File: Exchange/ExchangeServer/recipients/disconnected-mailboxes/connect-disabled-mailboxes.md [hunk 3]
```diff
@@ -102,6 +102,21 @@ Connect-Mailbox -Identity "Corporate Shared Mailbox" -Database "Mailbox Database
...
+```
+
+If you do not know the name of the personal archive, you can view it in the Exchange Management Shell by running the following command. This example returns all personal archive mailboxes in mailbox database DB01.
+
+```PowerShell
+Get-MailboxDatabase "DB01" | Get-MailboxStatistics | Where {($_.DisconnectDate -ne $null) -and ($_.IsArchiveMailbox -eq $true)} | Format-Table DisplayName,MailboxGuid -AutoSize
+```
+
+> [!NOTE]
+> You can connect a personal archive mailbox to a different primary mailbox, not matching the previous owner mailbox. Ensure using the _AllowLegacyDNMismatch_ parameter to allow connecting the archive mailbox to a different primary mailbox.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/recipients/disconnected-mailboxes/connect-disabled-mailboxes.md]:
```suggestion
> You can connect a personal archive mailbox to a different primary mailbox, not matching the previous owner mailbox. Ensure that you are using the _AllowLegacyDNMismatch_ parameter to allow the connection of the archive mailbox to a different primary mailbox.
```

**Comment by mattpennathe3rd** [comment]
@Apoc70 Thank you very much for your contribution! We appreciate it. The updated article will be published in the near future.

**Pull request merged**